### DISCUSSION TITLE
Errore scrutinio

### DISCUSSION THREAD

**Discussion by Davide031** [iisgiua/giuaschool#686] [Richieste di aiuto] [answered]
Buonasera un docente ha aperto lo scrutinio finale però appena clicca dal lato coordinatore la parola scrutinio esce un errore quindi non riusciamo a chiuderlo come facciamo (Non riusciamo a firmare, a mettere voti ecc. con questo errore)

**Comment by Davide031** [comment]
Allego il log
[2024-04-09_19-25.log](https://github.com/iisgiua/giuaschool/files/14922017/2024-04-09_19-25.log)

**Comment by ghost** [comment]
È stato effettuato lo scrutinio del Primo Periodo?

Il giorno mar 9 apr 2024 alle ore 19:28 Davide031 ***@***.***>
ha scritto:

**Comment by Davide031** [comment]
Si, ma io ora mi riferisco al secondo 

Il mar 9 apr 2024, 21:01 robertopadovesi ***@***.***> ha
scritto:

**Comment by Davide031** [comment]
Come potrei risolvere dato che non riesco più ad accedere allo scrutinio dal lato coordinatore?

**Comment by trinko** [comment, marked as answer]
Ci sono attualmente degli errori nella procedura dello scrutinio finale, che sto sistemando. Per cui l a procedura dello scrutinio finale per ora non è utilizzabile.

Puoi cancellare tutti gli scrutini finali aperti nel modo seguente:
- apri PHPMyAdmin dal pannello di controllo di Altervista
- inserisci nella finestra SQL ed esegui il codice seguente

```
UPDATE gs_scrutinio SET stato='N' WHERE periodo='F';
```

**Comment by Davide031** [comment]
Grazie, grazie a questo messaggio sono riuscito a risolvere grazie ancora e arrivederci
